torch/nn/parallel/comm.py

```diff
@@ -235,3 +242,8 @@ def gather(tensors, dim=0, destination=None, *, out=None):
                 "'destination' must not be specified when 'out' is specified, but "
                 "got destination={}".format(destination))
         return torch._C._gather_out(tensors, out, dim)
+
+
+def _handle_complex(tensor):
```

Comment:
Should we move this helper to torch._utils? Seems like the other tensor helper functions are imported from there

Reply:
Sure!